countrymap.js

```diff
@@ -181,7 +181,7 @@ var countryToCodeMap = {
 'Qatar': 'QA',
 'Réunion': 'RE',
 'Romania': 'RO',
-'Russian Federation': 'RU',
```

Comment:
I just like the shorter name better. It also takes less space on screen!

Reply:
How about we do the same for some other countries?
`Moldova, Republic of` -> `Moldova`
`Iran, Islamic Republic of` -> `Iran`

That makes it non-ISO and needs verification, but it should be fine. Furthermore I'm replacing Google Chart (which has all these bugs) with Mapbox which will not have any troubles rendering 2-letter-ISO-codes (I hope so) so we're using English names for localization only.